libs/wordpress-plugin/plugin/trunk/server/block-api-endpoints.php

```diff
@@ -409,6 +409,10 @@ function get_all_block_protocol_entities()
   return $entities;
 }
 
+function block_protocol_json_encode(array $value)
+{
+  return json_encode($value, JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE);
```

Comment:
I added `JSON_UNESCAPED_UNICODE` for good measure. Not sure if it would cause issues as we're using the WP charset collation for the table which could be non-utf8?

Reply:
I tried adding some multibyte unicode characters with this off, and they didn't end up escaped in the db anyway. I wonder if they get unescaped again if stored in a db with a unicode-encoding charset? Could you try putting in some data and querying that contains multibyte unicode characters? If it works anyway it might be worth not adding this in case it messes things for dbs with non-utf8 charsets. But I don't know.

Reply:
I will try with some of the complicated emojis

Reply:
Seems we want this there. The demo here shows with/without the unescape option. TL;DW the escaped (default) causes the emoji to split up and become unqueryable (tested on mariadb 10.2.7)

With

https://user-images.githubusercontent.com/6988668/228780158-26b1b1b0-ea14-44f2-bffa-c861bc870a31.mp4



Without

https://user-images.githubusercontent.com/6988668/228780169-9636e2e8-398c-46b9-b8e4-5f23128a2299.mp4

Reply:
Good to have confirmed, thank you for testing this out!